admin/src/collections/Messages.ts

```diff
@@ -0,0 +1,39 @@
+import { buildProperties } from '@camberi/firecms';
+import { Message, MESSAGE_FIRESTORE_PATH } from '../../../shared/src/types';
+import { buildAuditedCollection } from './shared';
+
+export const messagesCollection = buildAuditedCollection<Message>({
+	name: 'Messages',
+	group: 'Messages',
+	path: MESSAGE_FIRESTORE_PATH,
+	icon: 'SupervisorAccountTwoTone',
+	description: 'Lists all messages for one recipient or user',
+	customId: true,
+	properties: buildProperties<Message>({
+		type: {
+			dataType: 'string',
+			name: 'Type',
+			enumValues: {
+				sms: 'SMS',
+				email: 'EMAIL',
+			},
+		},
+		sent_at: {
+			dataType: 'date',
+			name: 'Message Date',
+			mode: 'date',
+		},
+		content: {
+			dataType: 'string',
+			name: 'Content',
+		},
+		to: {
+			dataType: 'string',
+			name: 'To',
+		},
+		status: {
+			dataType: 'string',
+			name: 'Status',
```

Comment:
Shall we model this as an enum?

Reply:
I would argue against it. Depending on the provider and channel (SMS or EMail) different status might be relevant